### PR TITLE
Sound player

### PR DESCRIPTION
**Pull request by Koziuk-S**
Added sounds player. Default play first sounds. Sounds play right around when you are click button next more and more. Was added playing sounds progress-bar.

### CODE REVIEW

File: favorite-music-markup/helper.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var helper = (function () {
```
Comment (marzubov) [inline favorite-music-markup/helper.js]: no need to wrap function inside in iife
it should be 
`var helper = function () {};`
or
`(function () { var Helper = ...; return window.helper = new Helper(); });`

File: favorite-music-markup/scripts.js [hunk 2]
```diff
@@ -1,126 +1,213 @@
...
-  resolve(_soundsList);
-  reject(new Error('Some error message....'));
-});
-
-function SoundsList() {
-  this.audioPlayer = document.querySelector('#audioPlayer');
-  this.controlBtn = document.querySelector('.fm-control-btn');
-  this.currentAudio = null;
-  this.isPlay = false;
+function SoundsList(_player) {
```
Comment (marzubov) [inline favorite-music-markup/scripts.js]: no need to use _player for argument if you don't have anything else with same name inside
```js
function SoundList(player) {
  this.player = player;
}
```

**Comment by marzubov** [comment]
@Koziuk-S

File: favorite-music-markup/scripts.js (hunk 2 above)
Comment (Koziuk-S) [reply favorite-music-markup/scripts.js]: Done

File: favorite-music-markup/helper.js (hunk 1 above)
Comment (Koziuk-S) [reply favorite-music-markup/helper.js]: Dane. And I created a normal module.

**Pull request merged**